### ISSUE TITLE
[TextField] autoFocus label is not floated

### ISSUE THREAD

**Issue by maloguertin** [material-components/material-components-web-react#873]
## Bug report
The TextField component does not behave properly when its Input children has `autoFocus` prop.

### Steps to reproduce
```jsx
<TextField label="foo">
 <Input autoFocus/>
</TextField>
```
### Actual behavior
The input has the focus but the label is not floated, nor does it float uppon entering text.

### Expected behavior
The input should be floated.

### Screenshots
![Screenshot from 2019-05-21 14-29-20](https://user-images.githubusercontent.com/6407064/58121222-e1051280-7bd4-11e9-9fcc-c6bf83cbfeb1.png)

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web         | 2.2.0  |
| Browser         |  Chrome (latest)|
| Operating System | Ubuntu 19 |

### Possible solution
When using `autofocus="true"` instead of `autoFocus` it works, it might be because the adapter uses the attribute `autofocus="true"` and React does not add the attribute, it only calls `.focus()` on mount. 

Also doing something like:
```jsx
<TextField label="foo">
 <Input ref={ref=>ref.inputElement.focus()}/>
</TextField>
```
yields the same result. Probably because the `handleFocusChange` prop passed to the InputComponent only sets the isFocused state but doesn't trigger anything on the adapter?

**Comment by moog16** [comment]
@maloguertin please see stackBlitz https://stackblitz.com/edit/react-afnzkv?file=TextField.js. Let me know if this is still an issue for you. Also you said you're on MDC Web version 2.2.0. This is the MDC React repo. Also you're missing a `d` in `<TextField ...`.

Closing as its not an issue, but feel free to ping this issue. I will respond.

**Comment by maloguertin** [comment]
It works on stackBlitz but no matter what I do can't get it to work on my build. Also I said MDC Web because were using it alongside the MDC React Repo (I have version 0.12.1 for the textfield).

**Comment by maloguertin** [comment]
Also if I alt-tab twice to leave and come back to the page then the label is properly floated

**Comment by moog16** [comment]
created a fix targeted for rc0.13.0

**Comment by moog16** [comment]
Closed in #875
